src/main/java/org/spongepowered/api/service/permission/PermissionService.java

```diff
@@ -64,6 +64,11 @@
      * subject is at the root of all inheritance trees, above even subject type-specific
      * defaults, meaning it has the lowest priority when all other weighting is equal.
      *
+     * <p>Note: This data should be persisted, so plugins that add permissions to this subject
+     * must take care to not override permissions already set or modified. It is also
+     * recommended to use {@link Subject#getTransientSubjectData()} where possible to
+     * avoid persisting unnecessary data.
```

Comment:
I'm not so sure about the persisting part.
If you modify defaults then its customized and no longer a default. IMO its better to specify a `user` group that inherits from the `default` group and overwrite things there.
In addition to that on one hand you force the permission plugin to persist the data, but on the other hand you ask the plugin owner to set transient data only.

Missing `</p>`tag.